src/ui/zabapgit_css_common.w3mi.xml

```diff
@@ -8,20 +8,20 @@
     <WWWPARAMS>
      <RELID>MI</RELID>
      <OBJID>ZABAPGIT_CSS_COMMON</OBJID>
-     <NAME>fileextension</NAME>
-     <VALUE>.css</VALUE>
+     <NAME>mimetype</NAME>
```

Comment:
https://github.com/abapGit/abapGit/commit/089e04d7e4bc7b156b853b0bcc58faa57a24297c fixed the ordering, so this will cause another diff?

Reply:
@FreHu I commited the file again and now there are no changes in the pull request on it. So I assume that the sorting works.